docs/topics/install/upgrading.md

```diff
@@ -1,6 +1,36 @@
 # Upgrading Ambassador Edge Stack
 
-Since Ambassador Edge Stack's configuration is entirely stored in Kubernetes resources, no special process is necessary to upgrade Ambassador Edge Stack. If you're using the YAML files supplied by Datawire, you'll be able to upgrade simply by repeating the following `kubectl apply` commands.
+Since Ambassador Edge Stack's configuration is entirely stored in Kubernetes resources, no special process
+is necessary to upgrade Ambassador Edge Stack.
+
+First, determine the installation method by running some `kubectl` command:
+
+* If the following command returns some resources then you should
```

Comment:
We need to have examples of the output that one would see in yes and no cases. Not just "returns some resources": please give examples of what the output would look like in the yes and no cases.

Reply:
Done.